scan-car/src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import Navigation from './components/navigation';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <Navigation />
```

Comment:
By convention <App/> should be your top-most element, and it should be the only component rendered in index.js.
I suggest moving <Navigation/> into <App/>.